src/components/HeroFc/HeroFc.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box, Button, Grid, Paper, Typography } from '@material-ui/core';
+import useTexts from '../../hooks/useTexts';
+import { useStyles } from './HeroFc.styles';
+//import { arrayIndexingWithLength } from '../../utils/arrayIndexingWithLength';
+import { theme } from '../../utils/theme';
+
+const HeroFc: React.FC = () => {
```

Comment:
HeroForCompany - FC in react is more known as "functional component" so it might cause confusion down the road